### ISSUE TITLE
Leave comments at the top

### ISSUE THREAD

**Issue by lencioni** [trotzig/import-js#139]
Similar to #77, people might want to put some comments at the top of their files, above the imports block. It might be nice to leave comments up there instead of forcing them to be below the imports block.

**Comment by trotzig** [comment]
I actually came across an example in our own codebase last night:

```js
'use strict';
/* eslint strict: [2, "global"] */
/* global process:false, __dirname:false */
```

This should be fairly straightforward to fix. The commit that introduced support for keeping `'use strict'` at the top might be useful to look at: https://github.com/trotzig/import-js/commit/ff026c1

**Comment by lencioni** [comment]
We can come up with a pretty decent regex to match multiline comments, but this is definitely not going to be foolproof. This would definitely be better if we were operating on an AST.

**Comment by lencioni** [comment]
One edge case that I'm not sure we'll be able to solve for is if you have a comment at the top that is supposed to apply to one of the imports.

```js
'use strict';
// We are importing this because of...
import Foo from 'foo';
```

However, I'm not sure if that will actually be a problem in practice, since most imports are pretty self explanatory.

**Comment by lencioni** [comment]
Closed by e400215
